cmd/maya-exporter/app/command/command_test.go

```diff
@@ -0,0 +1,118 @@
+package command
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmd = &cobra.Command{
+		Short: "Collect metrics from OpenEBS volumes",
+		Long: `maya-exporter can be used to monitor openebs volumes and pools.
+It can be deployed alongside the openebs volume or pool containers as sidecars.`,
+		Example: `maya-exporter -a=http://localhost:8001 -c=:9500 -m=/metrics`,
+	}
+)
+
+func TestAddListenAddressFlag(t *testing.T) {
+	cases := map[string]struct {
+		input *cobra.Command
+		value string
+	}{
+		"Valid Command": {
```

Comment:
@AmitKumarDas no, because the flags are passed while running the binary which takes everything as string. If there is any invalid input then system itself throw some error.